build.gradle.kts

```diff
@@ -60,16 +53,64 @@ allprojects {
         options.isFork = true
         options.isFailOnError = true
     }
+
+    // Tests
```

Comment:
Cleaned up and reorganized the root project's build file.  Each configuration is now labeled, which should help other developers who seek a way to do the same.